### ISSUE TITLE
`0-bootstrap` step fails with `terraform-google-modules/bootstrap/google` module version 4.1.0

### ISSUE THREAD

**Issue by vovinacci** [terraform-google-modules/terraform-example-foundation#601]
### TL;DR

`0-bootstrap` step fails with the latest `terraform-google-modules/bootstrap/google` module version 4.1.0.

If pinning to `4.0.0` - it works just fine.

May be related to https://github.com/terraform-google-modules/terraform-google-bootstrap/pull/134 after https://github.com/terraform-google-modules/terraform-google-project-factory/blob/release-v11.3.0/modules/project_services/versions.tf change introduced in https://github.com/terraform-google-modules/terraform-google-project-factory/commit/dd9beb3da1bcc0ab2ac2a95c6ad7e238031ef33a.

### Expected behavior

Step should not fail.

### Observed behavior

```
$ terraform init
Initializing modules...
Downloading registry.terraform.io/terraform-google-modules/bootstrap/google 4.1.0 for seed_bootstrap...
- seed_bootstrap in .terraform/modules/seed_bootstrap
Downloading registry.terraform.io/terraform-google-modules/kms/google 1.2.0 for seed_bootstrap.kms...
- seed_bootstrap.kms in .terraform/modules/seed_bootstrap.kms
Downloading registry.terraform.io/terraform-google-modules/project-factory/google 11.3.0 for seed_bootstrap.seed_project...
- seed_bootstrap.seed_project in .terraform/modules/seed_bootstrap.seed_project
- seed_bootstrap.seed_project.budget in .terraform/modules/seed_bootstrap.seed_project/modules/budget
- seed_bootstrap.seed_project.gsuite_group in .terraform/modules/seed_bootstrap.seed_project/modules/gsuite_group
- seed_bootstrap.seed_project.project-factory in .terraform/modules/seed_bootstrap.seed_project/modules/core_project_factory
- seed_bootstrap.seed_project.project-factory.project_services in .terraform/modules/seed_bootstrap.seed_project/modules/project_services
- seed_bootstrap.seed_project.quotas in .terraform/modules/seed_bootstrap.seed_project/modules/quota_manager
- seed_bootstrap.seed_project.shared_vpc_access in .terraform/modules/seed_bootstrap.seed_project/modules/shared_vpc_access

Initializing the backend...

Successfully configured the backend "gcs"! Terraform will automatically
use this backend unless the backend configuration changes.

Initializing provider plugins...
- Finding hashicorp/null versions matching ">= 2.1.0"...
- Finding latest version of hashicorp/time...
- Finding hashicorp/google versions matching ">= 3.43.0, >= 3.50.0, < 5.0.0"...
- Finding hashicorp/random versions matching ">= 2.2.0"...
- Finding hashicorp/google-beta versions matching ">= 3.1.0, >= 3.43.0, >= 3.50.0, < 5.0.0"...
- Installing hashicorp/google v4.3.0...
- Installed hashicorp/google v4.3.0 (signed by HashiCorp)
- Installing hashicorp/random v3.1.0...
- Installed hashicorp/random v3.1.0 (signed by HashiCorp)
- Installing hashicorp/google-beta v4.3.0...
- Installed hashicorp/google-beta v4.3.0 (signed by HashiCorp)
- Installing hashicorp/null v3.1.0...
- Installed hashicorp/null v3.1.0 (signed by HashiCorp)
- Installing hashicorp/time v0.7.2...
- Installed hashicorp/time v0.7.2 (signed by HashiCorp)

[ ... omitted for brevity ... ]
```

```
$ terraform plan
╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/seed_bootstrap.kms/main.tf line 30, in resource "google_kms_crypto_key" "key":
│   30:   key_ring        = google_kms_key_ring.key_ring.self_link
│ 
│ This object has no argument, nested block, or exported attribute named "self_link".
╵
╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/seed_bootstrap.kms/main.tf line 48, in resource "google_kms_crypto_key" "key_ephemeral":
│   48:   key_ring        = google_kms_key_ring.key_ring.self_link
│ 
│ This object has no argument, nested block, or exported attribute named "self_link".
╵
╷
│ Error: Unsupported attribute
│ 
│   on .terraform/modules/seed_bootstrap.kms/outputs.tf line 19, in output "keyring":
│   19:   value       = google_kms_key_ring.key_ring.self_link
│ 
│ This object has no argument, nested block, or exported attribute named "self_link".
╵
```

### Terraform Configuration

```hcl
module "seed_bootstrap" {
  source                         = "terraform-google-modules/bootstrap/google"
  version                        = "~> 4.0"
  org_id                         = var.org_id
  folder_id                      = google_folder.bootstrap.id
  billing_account                = var.billing_account
  group_org_admins               = var.group_org_admins
  group_billing_admins           = var.group_billing_admins
  default_region                 = var.default_region
  org_project_creators           = var.org_project_creators
  sa_enable_impersonation        = true
  parent_folder                  = var.parent_folder == "" ? "" : local.parent
  org_admins_org_iam_permissions = local.org_admins_org_iam_permissions
  project_prefix                 = var.project_prefix

  project_labels = {
    [ ... omitted for brevity ... ]
  }

  activate_apis = [
    [ ... omitted for brevity ... ]
  ]

  sa_org_iam_permissions = [
    [ ... omitted for brevity ... ]
  ]
}
```

### Terraform Version

```sh
1.0.10 - 1.1.0
```
